src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Neoscroll",
+  "description": "A smooth scrolling neovim plugin written in lua",
+  "link": "https://github.com/karb94/neoscroll.nvim",
+  "tags": [
+    "scrolling",
+    "lua"
```

Comment:
no need to add lua in there, scrolling should be enough!

Reply:
Then you should probably remove the "lua" tag from [template.json](https://github.com/lucax88x/nvim-awesome/blob/main/data/plugins/template.json) or other people will add it too.

Reply:
I totally agree!

Reply:
Updated the original template.

Reply:
You should also remove it from [CONTRIBUTING.md](https://github.com/lucax88x/nvim-awesome/blob/main/data/plugins/CONTRIBUTING.md)

Reply:
Right, fixed that part and removed the template. not useful there.